WebArchiveExtractor.xcodeproj/project.pbxproj

```diff
@@ -358,6 +358,7 @@
 				INFOPLIST_FILE = Info.plist;
 				INSTALL_PATH = "$(HOME)/Applications";
 				MACOSX_DEPLOYMENT_TARGET = 10.13;
+				OTHER_CODE_SIGN_FLAGS = "--deep";
```

Comment:
I don't know what that does - I did a quick google search and it's not super clear, can you let me know?

Reply:
honestly i dunno, all I remember is that the project would not build in Xcode, there was some recommendation given by Xcode to fix the issue and I hit apply, and then it built (and then these edits showed up in the `git diff`)

- are you able to build with this change reverted? 
- are you able to build with `xcodebuild` from the command line? (its possible I made this change to fix command line only builds)

Reply:
I don't think it's a problem, I was just curious about it. Once it's merged and runs though the github action, if its an issue I'll have a deeper look.